releases/22.10.0/22100 feature list.md

```diff
@@ -27,9 +29,11 @@ The general theme of this release is "TBD".
 * The Project Manager now displays the engine name and version number each project is registered with. The Project Manager now displays the current engine version in the title bar.
 * Script Canvas Architecture update that provides a framework to embed ScriptCanvas functionality in places other than Entity / Component system via the ScriptCanvas Component. The new architecture introduces a set of classes that handle a small part of the ScriptCanvas runtime. The highest level class is the ScriptCanvas::Executor which is now used by the ScriptCanvas::RuntimeComponent. The ScriptCanvas::Executor is also used by the new ScriptCanvasEditor::Interpreter, which is a class that allows developers to embed user access to ScriptCanvas functionality any where in the editor.
 * Writing C++ Script Canvas nodes is now a lot easier! We have deprecated node generics (a set of C++ macros) in favor of using AzAutoGen to produce libraries of functions or standalone nodes. We have consolidated the autogen semantics for Script Canvas grammar nodes and nodeables. We removed the concept of Script Canvas node libraries, anytime you write a node it gets registered automatically. It is no longer necessary to manually reflect or register Script Canvas nodes.
+* Prefab Edit Mode: Now when you double-click on a prefab you now go into prefab edit mode.
```

Comment:
update: This was technically in the previous release, we just didn't have great feedback. I'd suggest this instead be:

```
* Prefab Edit Mode Visual Feedback: Now when you double-click on a prefab and enter prefab edit mode, there is the option to enable an effect to grey out all entities not inside the current prefab.
```

Reply:
thanks for the feedback. updated this with your suggestion

Reply:
For future context @hultonha, My hope would be that we can implement double click to go into prefab edit mode in both the viewport and in the outliner. Last tested, this behavior only worked in the outliner. Not sure if we should make a ticket for this part as post release work. @AMZN-daimini let me know if I got that wrong.